### ISSUE TITLE
User signs up / logs in with Touchstone

### ISSUE THREAD

**Issue by pdpinch** [mitodl/open-discussions#771]
related to User story #770 

### acceptance criteria:
- [x] #671 create account with Touchstone
- [x] #672 recommend users with MIT email login with Touchstone
- [x] #763 set up SAML environment variables
- [x] create x509 certificates for CI, RC & Prod
- [x] register services with Touchstone folks at IS&T
- [ ] after authentication, user should be asked to confirm (or edit) their Display Name as returned by Touchstone,

**Comment by pdpinch** [comment]
I move this out of scope for this issue: after authentication, user should be asked to confirm (or edit) their Display Name as returned by Touchstone,

We can close this after #895 merges.
